Doppler.HtmlEditorApi/Controllers/CampaignsController.cs

```diff
@@ -69,6 +69,23 @@ public Task<ActionResult> GetCampaignThumbnail(string accountName, int campaignI
         [HttpPut("/accounts/{accountName}/campaigns/{campaignId}/content")]
         public async Task<IActionResult> SaveCampaign(string accountName, int campaignId, CampaignContent campaignContent)
         {
+            var campaignState = await _campaignContentRepository.GetCampaignState(accountName, campaignId);
+            if (!campaignState.OwnCampaignExists)
+            {
+                return new NotFoundObjectResult(new ProblemDetails()
+                {
+                    Title = $@"The campaign with id {campaignId} does not exists or belongs to another user than {accountName}"
```

Comment:
Should it be the title or the detail?

Reply:
I thought in this like a title because the text doesn't determinant, in this case could be that campaign no exist, the user no exist or the campaign belongs to another user. But you right, the text doesn't determinand but contain enoug information.

Maybe the correct message would be like this.
<pre>
{
  Title = $@"The campaign was no found",
  Detail = $@"The campaign with id {campaignId} does not exists or belongs to another user than {accountName}"
}
</pre>

What do you think?

Reply:
Nice!